src/main/scala/gitbucket/core/controller/IssuesController.scala

```diff
@@ -239,7 +240,7 @@ trait IssuesControllerBase extends ControllerBase {
       case (owner, name) =>
         getComment(owner, name, params("id")).map { comment =>
           if (isEditableContent(owner, name, comment.commentedUserName)) {
```

Comment:
Therefore, `isEditableContent` might not be not enough?

Reply:
@takezoe You mean we need something like `isDeletableContent`?? 

Acutually this ajaxPost endpoint is not limited as on UI. So means.,,
On ajaxPost Endpoint: developer role can delete comment.
On UI: developer role can not see delete button.
-> finally developer role can not delete comment.

Reply:
> this ajaxPost endpoint is not limited as on UI.

I see... So, this pull request is just willing to hide a delete button on UI. I think it's better to make those consistent. Let me check who calls this ajax endpoint before concluding this topic.

Reply:
Looks like this ajax endpoint is called from only the delete button.

https://github.com/gitbucket/gitbucket/blob/08ed3c4171373d8f6bad2c8ddbb7b393c6fc309e/src/main/twirl/gitbucket/core/issues/commentlist.scala.html#L318-L328

This means we can restrict this endpoint for only who can see the delete button?

Reply:
@takezoe I introduced `isDeletableComment` function at https://github.com/gitbucket/gitbucket/pull/2588/commits/be640d9539dcd964fa6c51e46dae62e1b12fc82c. Now UI and delete ajaPost are consistent.

Reply:
Thanks!!